libs/SmartStore/SmartStore/Classes/SFSmartStoreDatabaseManager.m

```diff
@@ -182,13 +182,9 @@ + (FMDatabase *)openDatabaseWithPath:(NSString *)dbPath key:(NSString *)key erro
 + (FMDatabase*) setKeyForDb:(FMDatabase*) db key:(NSString *)key error:(NSError **)error {
     [db setLogsErrors:YES];
     [db setCrashOnErrors:NO];
-    FMDatabase* unlockedDb = [self unlockDatabase:db key:key kdfIter:4000];
     
-    if (!unlockedDb) {
-        // You must be have created your database with SDK 3.2 or 3.1 and cocoapods
-        // And therefore you are using sqlcipher 3.1 with 64000 iterations
-        unlockedDb = [self unlockDatabase:db key:key kdfIter:64000];
```

Comment:
When we started using SQLCipher 3.x (in SDK 3.1), we didn't put in the code to keep using kdf iter of 4000 right away. Apps using the compiled library should be okay because we compiled with that default value - but apps using cocoapods ended up using the new kdf iter of 64000.
We fixed that a few months later. 
That code was there to gracefully handle opening those databases.
It's been 4 years. It's unlikely there are running apps using a database created in that short window. So dropping the code to support that case.

Reply:
Ah yes, I remember this. The difference was based on the way it was being consumed, i.e. Cocoapods had a different kdf iter.